aip/client-libraries/4231.md

```diff
@@ -219,6 +219,13 @@ to the following backwards-compatibility expectations:
   libraries.
 - The addition of a `google.api.resource_reference` annotation on an existing
   field **must** be a backwards-compatible change.
+- Changing a `google.api.resource_reference` from `child_type` to `type`
+  **must** be a backwards-compatible change when the `child_type` referenced
+  is single-parented and the newly referenced `type` is that exact parent.
+- Changing a `google.api.resource_reference` from `type` to `child_type`
```

Comment:
Thinking about it, this one *might* not be compatible - but I'll test and let you know. (It might depend on the name of the field, somewhat bizarrely... I'll explain how if and when we get there.)